src/tsc-bulk.ts

```diff
@@ -119,6 +119,14 @@ export function findDiagnosticsScopeId(d: DiagnosticTsc, isStrictScope: boolean)
   return `.${results.join('.')}`;
 }
 
+// if stdout is being piped to another destination, print diagnostics
+// without pretty colours. This matches the behaviour of TSC:
+// https://github.com/microsoft/TypeScript/blob/0652664e/src/compiler/executeCommandLine.ts#L161
```

Comment:
Thanks for providing this reference 🙏

Reply:
Strange that they tsc considers `NO_COLOR=0` to mean no color, but when in Rome 🤷‍♂️